sdk/translation/ai-translation-text-rest/src/customClient.ts

```diff
@@ -1,102 +0,0 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
This file needs to stay - it contains our custom logic.

Reply:
Restored.